compose.yaml

```diff
@@ -40,6 +40,8 @@ services:
         "
     volumes:
       - ./alfresco/resources/openannotate-override-placeholders.properties:/usr/local/tomcat/shared/classes/openannotate-override-placeholders.properties
+      - ./alfresco/resources/opencontent-override-placeholders.properties:/usr/local/tomcat/shared/classes/alfresco/module/com.tsgrp.opencontent/opencontent-override-placeholders.properties
```

Comment:
What is the property in this file that has been added to enable transformations?

Reply:
annotation.validDisplayableMimetypes=application/pdf
This is override property that can be added for this particular case. But in other cases few other AEV related properties which may be needed to be overriddeen against OC  can be done through this file.

Reply:
Where is the `annotation.validDisplayableMimetypes=application/pdf`?

I don't see this property in `opencontent-override-placeholders.properties` file

Reply:
It would be updated in the override property when needed by any team. Otherwise default it should not be in Properties file.